<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>字符串与数值之间的转换</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>数量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="suffix">
       <string>kg</string>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>单价</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="prefix">
       <string>$</string>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>qDdbug的測試</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>16進制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="maximum">
         <number>899999999</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>進行轉換</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>10進制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>進行轉換</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>二進制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="displayIntegerBase">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>進行轉換</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,3">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>总价</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="specialValueText">
       <string>0</string>
      </property>
      <property name="prefix">
       <string>$</string>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
